app/templates/courses/teacher-class-view.jade

```diff
@@ -140,7 +140,21 @@ mixin courseProgressTab
           span.small(data-i18n="teacher.level_not_started")
         .clearfix
     - var course = state.get('selectedCourse');
-    - var courseInstance = state.get('selectedCourseInstance');
+    - var courseInstance = view.getSelectedCourseInstance()
+    if me.showCourseProgressControll() && courseInstance && course && view.campaignLevelNumberMap[course.get('campaignID')]
+      .text-center(data-course-id=course.id)
+        span(data-i18n="courses.set_start_locked_level")
+        span.spr :
+        select.level-select
+          option(data-i18n="courses.no_block_level" value='none' selected=('none' === courseInstance.get('startLockedLevel')))
```

Comment:
I don't think this returns `'none'` when no level is locked. Should this be checking equality with `undefined`?

It may work just because this is the first option in the list as well (in which case I'd recommend removing the `selected` attribute)